other/upgrade_2-0_postgresql.sql

```diff
@@ -918,192 +918,67 @@ if (!isset($modSettings['attachment_thumb_png']))
 ---#
 
 /******************************************************************************/
---- Installing new default theme...
+--- Cleaning up after old themes...
 /******************************************************************************/
 
----# Installing theme settings...
+---# Checking for "babylon" and removing it if necessary...
 ---{
-// This is Grudge's secret "I'm not a developer" theme install code - keep this quiet ;)
-
-// Firstly, I'm going out of my way to not do this twice!
-if ((!isset($modSettings['smfVersion']) || $modSettings['smfVersion'] <= '2.0 RC2') && empty($modSettings['dont_repeat_theme_core']))
+// Do they have "babylon" installed?
+if (file_exists($GLOBALS['boarddir'] . '/Themes/babylon'))
 {
-	// Check it's not already here, just in case.
+	$babylon_dir = smf_mysql_real_escape_string($GLOBALS['boarddir'] . '/Themes/babylon');
 	$theme_request = upgrade_query("
-		SELECT id_theme
+		SELECT ID_THEME
```

Comment:
The sql keywords are commonly in uppercase and
the other stuff in lower case -> would keep the lowercase version

Reply:
albertlast - I am keeping things consistent with these other .sql files, and have copied this over from what has been in production use for quite some time.  I don't want to start making cosmetic changes at this point.